profiles/api/api.py

```diff
@@ -0,0 +1,54 @@
+import json
```

Comment:
generic name for a `api` module, could have been `profiles.api.profiles` (which is even more confusing)

Reply:
Could go the whole hog and call it `profiles.api.profiles_api` in case it implements a second endpoint (unlikely?).

Though maybe `profiles.api` isn't the best name.